programs/account-compression/src/instructions/initialize_address_merkle_tree.rs

```diff
@@ -42,6 +46,45 @@ pub fn process_initialize_address_merkle_tree<'info>(
                 .map_err(|_| AccountCompressionErrorCode::IntegerOverflow)?,
         )
         .map_err(ProgramError::from)?;
+    let init_value = BigUint::from_str_radix(
```

Comment:
I think adding the `FIELD_SIZE - 1` element should rather belong to the core `merkle-tree/indexed/src/lib.rs` code, in the `init` method. We already append the "0" low element there. Adding the "FIELD -1" high element here feels inconsistent.

Reply:
I have crated an init method